### PR TITLE
Reformat the source code

### PR DESCRIPTION
**Pull request by HaozhouWang**
Use tab to instead of space

### CODE REVIEW

**Comment by HaozhouWang** [comment]
This PR is related to https://github.com/greenplum-db/plcontainer/pull/187

File: rcall.c [hunk 1]
```diff
@@ -79,12 +79,12 @@
...
+            "{.Call(\"plr_SPI_prepare\", sql, argtypes)}"
 
 #define SPI_EXECP_CMD \
-			"pg.spi.execp <-function(sql, argvalues = NA) " \
-			"{.Call(\"plr_SPI_execp\", sql, argvalues)}"
+            "pg.spi.execp <-function(sql, argvalues = NA) " \
+            "{.Call(\"plr_SPI_execp\", sql, argvalues)}"
 
 #define PG_LOG_DEBUG_CMD \
     "plr.debug <- function(msg) {.Call(\"plr_debug\",msg)}"
```
Comment (paul-guo-) [inline rcall.c]: Why the three define are different?

File: rcall.c [hunk 2]
```diff
@@ -816,19 +828,19 @@ static void pg_get_one_r(char *value, plcDatatype column_type, SEXP *obj, int el
...
+		fldvec;
 
 	uint32 i, j;
 	int res = 0;
 
 	char buf[256];
 
-receive:
-    res = plcontainer_channel_receive(plcconn_global, &resp, MT_CALLREQ_BIT|MT_RESULT_BIT);
+	receive:
```
Comment (paul-guo-) [inline rcall.c]: No need of tab for such cases?

**Comment by HaozhouWang** [comment]
Related to https://github.com/greenplum-db/plcontainer/issues/88

**Pull request merged**